libs/s25main/Game.cpp

```diff
@@ -48,8 +49,9 @@ void Game::Start(bool startFromSave)
         if(ggs_.objective == GameObjective::EconomyMode)
         {
             unsigned int selection = ggs_.getSelection(AddonId::ECONOMY_MODE_GAME_LENGTH);
-            world_.econHandler = std::make_unique<EconomyModeHandler>(
-              std::chrono::minutes(AddonEconomyModeGameLengthList[selection]) / GAMECLIENT.GetGFLength());
+            world_.econHandler =
+              std::make_unique<EconomyModeHandler>(std::chrono::minutes(AddonEconomyModeGameLengthList[selection])
+                                                   / (std::chrono::milliseconds)SPEED_GF_LENGTHS[referenceSpeed]);
```

Comment:
```suggestion
                                                   / std::chrono::milliseconds(SPEED_GF_LENGTHS[referenceSpeed]));
```

Slightly better as it constructs instead of casts consistent with the other usages